src/components/SendFundsForm.vue

```diff
@@ -701,13 +703,16 @@ export default {
      */
     onScanQrcode(uri) {
       const recipient = parseURIasAIP(uri)
```

Comment:
I suggest to create a separate function for parsing KLY based URIs. For example:

```ts
function parseKlyURI(uri: string) {
  // ...
  return { address, amount,  }
}
```

Then you can create a function that will unify the result of both parsers:

```ts
function parseURI() {
  if (isKlyUri) {
    return parseKlyURI()
  }

  return parseURIasAIP() // ADM based URI
}
```